### ISSUE TITLE
IntegrationError on large screen size

### ISSUE THREAD

**Issue by etairi** [stripe/react-stripe-elements#447]
I'm using `react-stripe-elements` version "^5.0.0". I'm loading Stripe asynchronously, and using the `createPaymentMethod` with `CartElement` component. The interesting behavior I noticed is that when I run my app in mobile view (or other smaller screen sizes) it works all fine, however, when I switch to full screen desktop view, then it fails with the following error:

> Uncaught (in promise) IntegrationError: We could not retrieve data from the specified Element.
>               Please make sure the Element you are attempting to use is still mounted.

Is this a known bug, or any idea why this might happen?

**Comment by shivasurya** [comment]
@etairi  please let me know if you've found solution for this issue

**Comment by shivasurya** [comment]
@etairi  I've resolved the issue. Thank you.
for peoples who come searching for this issue,
In Angular 8.x version, I had to move the element card creation and mounting to ngAfterViewInit and verified that card element doesn't have any child elements using `cardElement.hasChildNodes` (verifies that it contains stripe iframe ) before mounting the card element in DOM. If you have any queries, do let me know. ( It took me 8 hours to try out a lot of solutions and understanding )
